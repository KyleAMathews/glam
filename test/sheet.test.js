/* eslint-env jest */
const {sheet, flush} = require('../src')

describe('sheet', () => {
  beforeEach(flush)

  test('insert', () => {
    const idx = sheet.insert('#bar { color: red; }')
    expect(idx).toBeDefined()
    console.log(sheet.rules())
    expect(sheet.rules()).toMatchSnapshot()
  })

  test('insert ssr', () => {
    flush()
    const old = global.window
    global.window = undefined

    const idx = sheet.insert('#bar { color: blue; }')
    expect(idx).toBeDefined()
    expect(sheet.rules()).toMatchSnapshot()
    console.log(sheet.rules())
    global.window = old
  })
})
