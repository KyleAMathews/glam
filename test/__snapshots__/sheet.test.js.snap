// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sheet insert 1`] = `Array []`;

exports[`sheet insert ssr 1`] = `Array []`;
